helm_deploy/manage-my-prison/values.yaml

```diff
@@ -22,9 +22,8 @@ generic-service:
     enabled: true
     host: app-hostname.local    # override per environment
     tlsSecretName: cert
-    path: /
+    contextColour: green
```

Comment:
need to specify the colour for the `generic-service` - it build build up the rest of the annotation